### ISSUE TITLE
Maps API and Overlays

### ISSUE THREAD

**Issue by ArchangeGabriel** [microg/android_packages_apps_GmsCore#238]
Hi,

Two apps I’m using are having troubles with Maps API (or maybe something else): OpenSignal and Sensorly.

Both are supposed to show an overlay of signal coverage/strength above the world map provided by the Maps API. But with μG, they only show the map. Also, both are supposed to display current location and follow it if I’m travelling, but except that I can see in status bar a that location is being requested and obtained by the GPS, nothing happens in the apps.

And finally, Sensorly stops responding on any move, zoom in or out, etc.

Please tell me if there is anything I can provide or test for you to fix this. ;)

**Comment by ale5000-git** [comment]
Verify that you have installed lates version, currently 0.2.4-**30**.

**Comment by ArchangeGabriel** [comment]
Yes, I use this exact version. In fact it’s even @mar-v-in latest comment on #236 that made me check with this new version and report. ;)

**Comment by axydavid** [comment]
The latest patch shouldn't help you, as far as I see in thee code, he explicitly changed it so ONLY the housenumbers, roads and cities that will overlay the line.

**Comment by ArchangeGabriel** [comment]
Well, it didn’t worked before anyway. So could this issue be that app’s overlays get rendered below the map?

**Comment by mar-v-in** [comment]
No, the map overlay is not rendered at all. This feature is missing from the current Maps API implementation.

**Comment by ArchangeGabriel** [comment]
Oh, OK, I see. :) So this part is a missing feature. Given the status of Maps API in μG currently, I suppose that the crashes and location issues are to be expected?
